### PR TITLE
Feature/advanced configuration update

### PR DESCRIPTION
**Pull request by HonzaCuhel**
This PR includes the following new pipeline arguments:

- `--annotation_iou_threshold`: Intersection over Union (IoU) threshold for annotation.
- `--prompt_prefix`: Prefix to add to every image generation prompt.
- `--prompt_suffix`: Suffix to add to every image generation prompt, e.g., for adding details like resolution.
- `--negative_prompt`: List of of negative prompts to guide the generation away from certain features.
- `--annotator_size`: Size of the annotator model to use. Choose between `base` and `large`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,30 +101,35 @@ datadreamer --save_dir <directory> --class_names <objects> --prompts_number <num
...
-- `--conf_threshold`: Confidence threshold for annotation. Default is 0.15.
-- `--use_tta`: Toggle test time augmentation for object detection. Default is True.
+- `--conf_threshold`: Confidence threshold for annotation. Default is `0.15`.
+- `--annotation_iou_threshold`: Intersection over Union (IoU) threshold for annotation. Default is `0.2`.
+- `--prompt_prefix`: Prefix to add to every image generation prompt. Default is `""`.
+- `--prompt_suffix`: Suffix to add to every image generation prompt, e.g., for adding details like resolution. Default is `hd 8k "highly detailed"`.
+- `--negative_prompt`: List of of negative prompts to guide the generation away from certain features. Default is `cartoon "blue skin" painting scrispture golden illustration "worst quality" "low quality" "normal quality:2" "unrealistic dream" "low resolution" static "sd character" "low quality" "low resolution" greyscale monochrome nose cropped lowres "jpeg artifacts" "deformed iris" "deformed pupils" "bad eyes" "semi-realistic worst quality" "bad lips" "deformed mouth" "deformed face" "deformed fingers" "bad anatomy"`.
```
Comment (sokovninn) [inline README.md:121]: Why do some words here have quotes and some do not? And isn't it a bit confusing to call it a list when it's a string?

**Review by kozlov721 — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (HonzaCuhel) [reply README.md:121]: I decided to quote the flag names and the argument values (including the default values) to separate them from normal text. However, I am open to discussion; if you deem it confusing, I'll change it.

Comment (HonzaCuhel) [reply README.md]: Updated!

Comment (sokovninn) [reply README.md]: Thanks, looks good!

**Pull request merged**